src/staging/src/utils/helpers.js

```diff
@@ -30,3 +30,11 @@ export const renderBreadcrumbs = (menuContent, pageContext) => {
         return breadcrumbItems.reverse()
     }
 }
+export const addPrefix = (str) => {
```

Comment:
Since this is a generic helpers file, the method name should be more descriptive.  e.g. `addSmallImagePrefix`